pakyow-core/lib/pakyow/core/controller.rb

```diff
@@ -35,12 +35,14 @@ def process
         catch :halt do
           hook_around :route do
             app.state_for(:router).each do |router|
+              @processing = router
               @found = router.call(
                 request.env[Rack::PATH_INFO],
                 request.env[Rack::REQUEST_METHOD],
                 request.params,
                 context: self
               )
+              @processing = nil
```

Comment:
This is just to clear the "router being processed", so we can be sure that the router didn't cause an error. 

I know its not likely that the internals would cause an error, but in the case that there is some bug, then we don't necessarily want the exception handling to assume that the router caused the error (which it would if `@current_router` is the last router processed).

An alternative, which might be more dynamic is to update the `failed_router` method to be:

```ruby
def failed_router
  !found && @current_router
end
```